### PR TITLE
Use Razor Sdk from CLI

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/UI/Microsoft.AspNetCore.Identity.UI.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk;Microsoft.NET.Sdk.Razor">
```
Comment (rynowak) [inline src/UI/Microsoft.AspNetCore.Identity.UI.csproj]: You shouldn't need both of these right? Or did we forget to include the net sdk 😢

File: src/UI/Microsoft.AspNetCore.Identity.UI.csproj [hunk 2]
```diff
@@ -10,6 +10,7 @@
     <GenerateEmbeddedFilesManifest>true</GenerateEmbeddedFilesManifest>
     <RazorCompileOnBuild>true</RazorCompileOnBuild>
     <ResolvedRazorCompileToolset>RazorSdk</ResolvedRazorCompileToolset>
```
Comment (rynowak) [inline src/UI/Microsoft.AspNetCore.Identity.UI.csproj]: Is this still required? We might already have a bug tracking that.

File: src/UI/Microsoft.AspNetCore.Identity.UI.csproj (hunk 1 above)
Comment (pranavkm) [reply src/UI/Microsoft.AspNetCore.Identity.UI.csproj]: Yeah, I guess we missed covering this in our take-over from Web.Sdk thing. We'd have to effectively move this from line from https://github.com/aspnet/websdk/blob/master/src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.props#L20 to Razor.Sdk

File: src/UI/Microsoft.AspNetCore.Identity.UI.csproj (hunk 2 above)
Comment (pranavkm) [reply src/UI/Microsoft.AspNetCore.Identity.UI.csproj]: It's hidden behind the flag that needs to be cleaned up. Need one more round of stuff to get in to the CLI to sort those out.

**Comment by rynowak** [comment]
Consider me signed off on this then. We shouldn't block on these issues, but we do need to fix them.

**Pull request merged**